.travis.yml

```diff
@@ -21,7 +21,17 @@ script: bash -ex .travis-ci.sh
 matrix:
   include:
   - env: CI_KIND=build XARCH=i386
-  - env: CI_KIND=build XARCH=i386 CONFIG_ARG=-flambda OCAMLRUNPARAM=b,v=0
+    addons:
+      apt:
+        packages:
+        - gcc:i386
+        - cpp:i386
+        - binutils:i386
+        - binutils-dev:i386
+        - libx11-dev:i386
+        - libc6-dev:i386
+  - env: CI_KIND=build XARCH=x64
+  - env: CI_KIND=build XARCH=x64 CONFIG_ARG=-flambda OCAMLRUNPARAM=b,v=0
```

Comment:
To be sure I understand: previously we were only testing Flambda in 32-bit mode and now we are only testing it in 64-bit mode?  (OK if so.)

Reply:
No, previously all the tests were 64-bit - XARCH saying i386 had no effect because the Travis vm is 64-bit. So the flambda test is unchanged. We could add another case for 32-bit flambda if you think that'd be useful for each PR?

Reply:
I think 64 bit only should suffice.